[esx_addons]/esx_basicneeds/server/main.lua

```diff
@@ -1,34 +1,28 @@
 CreateThread(function()
-	for k,v in pairs(Config.Food) do
+	for k,v in pairs(Config.Items) do
 		ESX.RegisterUsableItem(k, function(source)
 			local xPlayer = ESX.GetPlayerFromId(source)
 			if v.remove then
 				xPlayer.removeInventoryItem(k,1)
 			end
-			TriggerClientEvent("esx_status:add",source,"hunger",v.status)
-			TriggerClientEvent('esx_basicneeds:onEat',source)
-			xPlayer.showNotification(TranslateCap('used_eat', ESX.GetItemLabel(k)))
-		end)
-	end
-end)
-
-CreateThread(function()
-	for k,v in pairs(Config.Drinks) do
-		ESX.RegisterUsableItem(k, function(source)
-			local xPlayer = ESX.GetPlayerFromId(source)
-			if v.remove then
-				xPlayer.removeInventoryItem(k,1)
+			if v.type == "food" then
+				TriggerClientEvent("esx_status:add", source, "hunger", v.status)
+				TriggerClientEvent('esx_basicneeds:onUse', source, v.type)
+				xPlayer.showNotification(TranslateCap('used_food', ESX.GetItemLabel(k)))
+			elseif v.type == "drink" then
+				TriggerClientEvent("esx_status:add", source, "thirst", v.status)
+				TriggerClientEvent('esx_basicneeds:onUse', source, v.type)
+				xPlayer.showNotification(TranslateCap('used_drink', ESX.GetItemLabel(k)))
+			else
+				print(string.format('^1[ERROR]^0 %s has no correct type defined.', k))
```

Comment:
If the v.type is nil will it not register an item? Think about how this could be solved. It might be enough to throw an deprecated error message.

@Kuuzoo Any ideas for you?

Reply:
The Item is getting registered over the "k" variable not over v.type, but also if v.type is not defined it's going with a error print.
I could check if k is a string, but k is getting defined everytime by lua. So if it's not a string it's the index of the table.

So the print would only make sense if k is not a string.

Hope you understood what I meant.

Reply:
Sorry, I said it in a stupid way, so the item will be registered but I thought it's better this way. If you update it, it will use the new Config.Items anyway, no problem. 

No further changes are needed. 
I  merge tomorrow. 
Thank you very much for the pull request.

Reply:
Thank you for all the tips, your time and just everything ❤️.

Have a great night!
Best regards, kuuzoo!